packages/devtools_extensions/bin/_build_and_copy.dart

```diff
@@ -84,9 +84,6 @@ class BuildExtensionCommand extends Command {
 
     _log('Copying built output to the extension destination...');
     await _copyBuildToDestination(source: source, dest: destination);
-
-    // Closes stdin for the entire program.
-    await sharedStdIn.terminate();
```

Comment:
this should not be removed since this does not use the `DevToolsCommandRunner` from the `tool/` directory

Reply:
Good catch - fixed